tick/stateful/functions.go

```diff
@@ -12,21 +12,76 @@ import (
 
 	"github.com/dustin/go-humanize"
 	"github.com/influxdata/influxdb/influxql"
+	"github.com/influxdata/kapacitor/tick/ast"
 )
 
+const (
+	maxArgs = 4
```

Comment:
Can you add a comment here that this about the max args for any function? That way it is clear it needs to be increased if a new function is added.